The patch fix the test on aarch64.

It adjusts the default value of S2K_MINIMUM_TUNING_RATIO based on
the target architecture. Specifically, it lowers the minimum tuning
ratio from 4 to 3 on AArch64 (ARM 64-bit) systems, while keeping the
original value of 4 for all other architectures.
--- a/src/tests/cipher.cpp.orig
+++ b/src/tests/cipher.cpp
@@ -738,7 +738,11 @@
 // platforms known to not have a robust response can compile with
 // -DS2K_MINIMUM_TUNING_RATIO=2 (or whatever they need)
 #ifndef S2K_MINIMUM_TUNING_RATIO
-#define S2K_MINIMUM_TUNING_RATIO 4
+#  ifdef __aarch64__
+#    define S2K_MINIMUM_TUNING_RATIO 3
+#  else
+#    define S2K_MINIMUM_TUNING_RATIO 4
+#  endif
 #endif
 
 TEST_F(rnp_tests, s2k_iteration_tuning)
